subjects/devops/input-redirection/README.md

```diff
@@ -0,0 +1,43 @@
+## Input redirection
+
+### Instructions
+
+In this exercise you will make a script `input-redirection.sh`.
+This script will read from an here document (`HereDoc`).
+Usually this technique is used to programmatically generate scripts or configuration files receiving some multiline input.
+
+The script will create a file `show_info.sh` that will run the command `cat` with `-e` as argument.
```

Comment:
to keep the naming convention it might be better to rename `show_info.sh` to `show-info.sh`

Reply:
done